hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -97,6 +99,18 @@ def get_content_ref_details(self) -> t.Tuple[str, ContentRefType]:
         return (self.content_id, ContentRefType.DEFAULT_S3_BUCKET)
 
 
+@dataclass
+class SubmitContentHashRequestBody(SubmitRequestBodyBase):
+    signal_value: str
+    signal_type: str  # SignalType.getname() values
+    content_url: t.Optional[str]
```

Comment:
I'm assuming this is for the case where we have a content_url, but do not want to hash it again.

Love it!